specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2021-01-25/guestconfiguration.json

```diff
@@ -1571,12 +1571,12 @@
       "properties": {
         "vmId": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
           "description": "UUID of the VM."
         },
         "vmResourceId": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
           "description": "Azure resource Id of the VM."
         },
         "complianceStatus": {
```

Comment:
May I ask why the properties you changed in this PR are different in different API version? Per my understanding,  the properties you need to change should be same in different API version, right?

Reply:
As mentioned earlier - none of our properties are read-only.
Not sure why historically the swagger was designed to have these properties read-only.
So go forward trying to remove where ever possible.

Reply:
event if we keep the properties writable - end user wont be able to write because we are handling things in service side.